### ISSUE TITLE
 File integrity check failed plugins/Morpheus/icons/submodules

### ISSUE THREAD

**Issue by devent** [matomo-org/docker#64]
Hello, I just try out Piwik in Docker and I get the error from Piwik System Check.

```
File integrity check failed and reported some errors. You should fix this issue and then refresh this page until it shows no error.

Directories were found in your Piwik but we didn't expect them.
--> Please delete these directories to prevent errors. <--

Directory to delete: plugins/Morpheus/icons/submodules


To delete all these directories at once, you can run this command:
rm -Rf /var/www/html/plugins/Morpheus/icons/submodules
```

What is this Morpheus plugin? Because I can't find it on https://plugins.piwik.org/search?piwikversion=0&query=Morpheus

**Comment by bnason** [comment]
See piwik/piwik#11732 - File integrity check failed

**Comment by Findus23** [comment]
Hi, this will be fixed with Piwik 3.0.5.

> What is this Morpheus plugin? Because I can't find it on https://plugins.piwik.org/search?piwikversion=0&query=Morpheus

`Morpheus` is the name of the default theme.
